From 81c9833b810ba2d3e7351256ad4ee1eaba1bc2a3 Mon Sep 17 00:00:00 2001
From: Xinliang Liu <xinliang.liu@linaro.org>
Date: Mon, 28 Dec 2015 17:30:58 +0800
Subject: [PATCH 01/10] drivers/gpu: Add ARM Mali Utgard r6p0 driver

Add ARM Mali Utgard driver

The assumption is that the contents of directory driver/src/devicedrv/mali/
from DX910-SW-99002-r6p0-01rel0.tgz is unpacked into drivers/gpu/arm/utgard/
in the kernel source tree.

Signed-off-by: Xinliang Liu <xinliang.liu@linaro.org>
---
 drivers/gpu/Makefile                               |    2 +-
 drivers/gpu/arm/Kconfig                            |    1 +
 drivers/gpu/arm/Makefile                           |    1 +
 drivers/video/Kconfig                              |    1 +

---
 drivers/gpu/Makefile     |    2 +-
 drivers/gpu/arm/Kconfig  |    1 +
 drivers/gpu/arm/Makefile |    1 +
 drivers/video/Kconfig    |    1 +
 4 files changed, 4 insertions(+), 1 deletion(-)

--- a/drivers/gpu/Makefile
+++ b/drivers/gpu/Makefile
@@ -2,5 +2,5 @@
 # taken to initialize them in the correct order. Link order is the only way
 # to ensure this currently.
 obj-$(CONFIG_TEGRA_HOST1X)	+= host1x/
-obj-y			+= drm/ vga/
+obj-y			+= drm/ vga/ arm/
 obj-$(CONFIG_IMX_IPUV3_CORE)	+= ipu-v3/
--- /dev/null
+++ b/drivers/gpu/arm/Kconfig
@@ -0,0 +1 @@
+source "drivers/gpu/arm/utgard/Kconfig"
--- /dev/null
+++ b/drivers/gpu/arm/Makefile
@@ -0,0 +1 @@
+obj-y += utgard/
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -18,6 +18,7 @@ source "drivers/gpu/vga/Kconfig"
 source "drivers/gpu/host1x/Kconfig"
 source "drivers/gpu/ipu-v3/Kconfig"
 
+source "drivers/gpu/arm/Kconfig"
 source "drivers/gpu/drm/Kconfig"
 
 menu "Frame buffer Devices"
